src/utils/ezWallet.js

```diff
@@ -0,0 +1,119 @@
+import EventEmitter from 'eventemitter3';
```

Comment:
Please don't copy the whole wallet here. import it from ezDeFi project

Reply:
resolved